recipes/datacache/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "datacache" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "29bce433db1f379e1439ca640b59a8ebce617cf500667ebb48d70c1dc2c40cc5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - pandas >=0.15.2
+    - appdirs >=1.4.0
+    - progressbar33 >=2.4
+    - biopython >=1.65
+    - requests >=2.5.1
+    - typechecks >=0.0.2
+    - mock
+
+test:
+  imports:
+    - datacache
+
+about:
+  home: https://github.com/openvax/datacache
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
@mariusvniekerk , does this mean I need to remove the license file?

Reply:
Ideally make a comment above this line referencing the pull request to get that changed upstream

Reply:
Thanks for the explanation @mariusvniekerk . According to the conda-forge documentation, http://conda-forge.readthedocs.io/en/latest/meta.html, adding missing license this way is suggested. I am hesitant to open an upstream request for the license file since I don't know how long it will take for the developers to add one, and I would like this package available soon.